PostHog/PostHogStorage.swift

```diff
@@ -129,6 +119,39 @@ class PostHogStorage {
         setData(forKey: key, contents: data)
     }
 
+    /**
+     There are cases where applications using posthog-ios want to share analytics data between host app and an app extension, Widget or App Clip. If there's a defined `appGroupIdentifier` in configuration, we want to use a shared container for storing data so that extensions correcly identify a user (and batch process events)
+     */
+    private static func getAppFolderUrl(from configuration: PostHogConfig? = nil) -> URL {
```

Comment:
I think config does not need to be null here since we always call with non null values